benchmarks/halo_finder/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+if(Kokkos_DEVICES MATCHES "CUDA")
+  set(ArborX_TARGET ArborX)
+
+  # Separate ECL into a separate target. Otherwise, due to MPI linkage,
+  # it gets MPI_CXX_COMPILE_OPTIONS that has flags like `-fexceptions` which
+  # break nvcc.
+  add_library(ECL ECL-CC_10.cu)
```

Comment:
I think having it in its own target was appropriate regardless.
Did you ignore the header/include dirs on purpose?

Reply:
I guess I could add `ECL.hpp` here. Did not think much on it.